cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -146,39 +146,13 @@ def get_directory_service_dna_json_for_head_node(config: BaseClusterConfig) -> d
     )
 
 
-def get_shared_storage_ids_by_type(shared_storage_ids: dict, storage_type: SharedStorageType):
-    """Return shared storage ids from the given list for the given type."""
-    default = "" if storage_type == SharedStorageType.EFS else "NONE"
-    return (
-        ",".join(storage_mapping.id for storage_mapping in shared_storage_ids[storage_type])
-        if shared_storage_ids[storage_type]
-        else default
-    )
+def get_comma_separated_string(list):
```

Comment:
This might read better if it's called `to_comma_separated_string` because that's what it's producing

Reply:
Done